src/webgpu/api/operation/render_pipeline/culling_tests.spec.ts

```diff
@@ -99,17 +184,47 @@ TODO: check the contents of the depth and stencil outputs [2]
       colorAttachments: [
         {
           view: texture.createView(),
-          clearValue: { r: 0.0, g: 0.0, b: 1.0, a: 1.0 },
+          clearValue: [0, 0, 1, 1],
           loadOp: 'clear',
           storeOp: 'store',
         },
       ],
       depthStencilAttachment,
     });
 
-    // Draw two triangles with different winding orders:
-    // 1. The top-left one is counterclockwise (CCW)
-    // 2. The bottom-right one is clockwise (CW)
+    // Draw triangles with different winding orders:
+    //
+    // for triangle-list, 2 triangles
+    //   1. The top-left one is counterclockwise (CCW)
+    //   2. The bottom-right one is clockwise (CW)
+    //
+    //     0---2---+
+    //     |   |   |
+    //     |   |   |
+    //     1---+---4
+    //     |   |   |
+    //     |   |   |
+    //     +---3---5
+    //
+    // for triangle-strip, 4 triangles
+    //   1. The top left is counterclockwise (CCW)
+    //   2. zero size
+    //   3. zero size
+    //   4. The bottom right one is clockwise (CW)
```

Comment:
nit: Note in the comment the triangles are (I think) 012, 213, 234, 435 because of how triangle-strip works.
Initially I thought the last triangle was 345 which would have been CCW.

Reply:
added a comment on that